contracts/FragToken.sol

```diff
@@ -9,15 +9,24 @@ import "openzeppelin-solidity/contracts/access/Ownable.sol";
 
 contract FRAGToken is ERC20, ERC20Permit, Ownable {
     uint8 constant DECIMALS = 12; // Preferred for Fragnova (Substrate)
-    uint256 constant INITIAL_SUPPLY = 10_000_000_000 * (10**DECIMALS);
+    uint256 constant INITIAL_SUPPLY = 10_000_000_000 * (10**DECIMALS); 
+    uint256 private _lockCooldown = 1 weeks; // Roughly 1 week 
```

Comment:
can remove the comment 😄

Reply:
Done, I removed completely the cooldown variable and setter function.